rerun_py/rerun_sdk/rerun/blueprint/api.py

```diff
@@ -91,6 +91,28 @@ def __init__(
         self.defaults = defaults
         self.overrides = overrides
 
+    def __eq__(self, other: Any) -> bool:
+        """
+        Equality implementation, useful for tests.
+
+        This ignores the `id` field, which should always be unique.
```

Comment:
it's unique for a reason though - every view instance represents a different view in the viewer. Does it really make sense to call two views "equal" then? 🤔 
I see that this is obviously useful for testing, but it's not quite correct as presented I think

Reply:
You're obviously right. The thinking is that no one else in their right mind would test for view equality. That said, I just had an idea on a possible cheap way to avoid this.